test/passing/record.ml

```diff
@@ -106,3 +106,5 @@ let _ = x {a: string = a'; b= (b' : string)}
 let x = function {a= (_ : string); _} -> ()
 
 let x = function {a: string = _; _} -> ()
+
+let x = {aaaaaaaaaa (* b *); b}
```

Comment:
Can you add an example with a new line like 
```ocaml
let x =
  { aaaaaaaaaa
  (* b *)
  ; b}
```

Reply:
done